scripts/odm_georeferencing.py

```diff
@@ -117,9 +117,6 @@ def process(self, inputs, outputs):
             geo_ref = types.ODM_GeoRef()
             geo_ref.parse_coordinate_system(tree.odm_georeferencing_coords)
 
-            for idx, photo in enumerate(self.inputs.photos):
```

Comment:
This is probably the line to change to restore geotagging of resized photos if that's not currently working.